pod.go

```diff
@@ -191,8 +191,12 @@ func (pc *podCollector) collectPod(ch chan<- prometheus.Metric, p v1.Pod) {
 	addCounter := func(desc *prometheus.Desc, v float64, lv ...string) {
 		addConstMetric(desc, prometheus.CounterValue, v, lv...)
 	}
+	for _, containerStatus := range p.Status.Conditions {
+		if containerStatus.Type == "Initialized" {
+			addGauge(descPodInfo, 1, p.Status.HostIP, p.Status.PodIP, containerStatus.LastTransitionTime.String(), nodeName, extractCreatedBy(p.Annotations))
```

Comment:
According to [client-go](https://github.com/kubernetes/client-go/blob/master/pkg/api/types.go#L1684-L1688),
`Initialized` indicates that `all init containers in the pod have started successfully`. `Ready` represents that `the pod is able to service requests and should be added to the load balancing pools of all matching services'.